web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -8,22 +8,25 @@ axios.defaults.baseURL = defaultEndpointPrefix();
 // withCredentials ensures that axios fetch includes `HttpOnly` cookies in the request for CSRF protectioin
 axios.defaults.withCredentials = true
 
+const AUTH_SESSION_KEY = '__SESSION_TOKEN__';
 
 axios.interceptors.response.use(
     (response) => {
         return response;
     },
-    (error) => {
+    async (error) => {
         let message;
 
         if (error && error.response && error.response.status === 404) {
             // window.location.href = '/not-found';
         } else if (error && error.response && error.response.status === 403) {
-            window.location.href = '/access-denied';
+            sessionStorage.removeItem(AUTH_SESSION_KEY)
+            window.location.href = '/login';
         } else {
             switch (error.response.status) {
                 case 401:
                     message = 'Invalid credentials';
+                    window.location.href = '/login'
```

Comment:
is `sessionStorage.removeItem(AUTH_SESSION_KEY)` also required here?

Reply:
No, but I'm testing the way to fix an issue about redirection when user token expires